users/details/style.css

```diff
@@ -438,3 +438,6 @@ footer p a {
   display: flex;
   justify-content: center;
 }
+.not-found-img {
+  width: 55%;
```

Comment:
Also, add max-width style property for the image.

Reply:
added